docs/migrate-module.md

```diff
@@ -0,0 +1,94 @@
+# Update an older module
+
+Module repos created before July 2022 don't include the logic that is needed to pass the continuous integration (CI) tests and to publish the modules to the IBM Cloud catalog. Follow these steps to update an existing repo in the terraform-ibm-modules GitHub organization.
+
+## Before you begin
+
+- If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
+- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a recent stable version of [Go](https://go.dev/doc/install) installed and available in your PATH environment variable.
+
+## Create an empty module
+
+Create an empty module repo in the `terraform-ibm-modules` GitHub organization.
+
+1.  In the upper right of any page in the [terraform-ibm-modules](https://github.com/terraform-ibm-modules) organization, use the **+** menu, and select **New repository**.
```

Comment:
Only admins can create repos, I think we'll need to design a mechanism to request a new module repo to be created - eg: an issue at https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker with a specific label "module-onboarding"

Reply:
why would we be documenting how to create an empty repo? Don't we always want people to use the module template?

Reply:
@ocofaigh I had that same thought about the template. See the comment in the related internal issue 2357. Will sync up with both of you about proper direction for this.

Reply:
Left it as an empty module. I'll create a PR for a new issue template. 
Let me know if we want to use the template instead.

Reply:
@ocofaigh - That's one approach, but then the exercise it about resolving conflicts between existing code base and the template in main. That does not sound appealing. We started with minimal repo for landing-zone for instance. https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/commit/e548d6f5413a8e25a8ae9ab9eddee846ca78b2de

Reply:
Leaving this as a blank repo for now. Will create a PR in the `.github` repo for a new repo issue template.